tests/zenko_tests/e2e-config.yaml

```diff
@@ -1,8 +1,25 @@
 accounts:
   - "zenko"
-endpoints: []
-locations: []
+endpoints:
+  - hostname: "http://aws-mock"
+    locationName: "awsbackendmismatch"
+  - hostname: "http://azure-mock/devstoreaccount1"
+    locationName: "azurebackendmismatch"
```

Comment:
the endpoints seems incorrect.
`endpoints` is the list of hostnames the `s3api` can be reached by

the mock service endpoints should exist when they are deployed